install.md

```diff
@@ -513,3 +513,36 @@ sudo systemctl start crio
 - Follow this [tutorial](tutorials/crictl.md) to quickly get started running simple pods and containers.
 - To run a full cluster, see [the instructions](tutorials/kubernetes.md).
 - To run with kubeadm, see [kubeadm instructions](tutorials/kubeadm.md).
+
+### Updating CRI-O
+
+#### openSUSE:
+```
+sudo zypper update
+sudo zypper update cri-o
+```
+
+### Fedora 31 or later
+```
+sudo dnf update
+sudo dnf update cri-o
+```
+
+### APT based operating systems
+be sure that the environment variable ```$OS``` is set to your operating system
```

Comment:
Would be nice to give an example here.

Reply:
does my update fix this or is it too much ?